src/EPR.Calculator.API/Constants/CalcResultSummaryHeaders.cs

```diff
@@ -39,14 +39,14 @@ public static class CalcResultSummaryHeaders
         public static readonly string NorthernIrelandTotalwithBadDebtprovision = "Northern Ireland Total with Bad Debt provision";
 
         public static readonly string TotalProducerDisposalFee = "1 Total Producer Disposal Fee w/o Bad Debt Provision";
-        public static readonly string TotalProducerDisposalFeeWithBadDebtProvision = "1 Total Producer Disposal Fee with Bad Debt Provision";
```

Comment:
This is incorrect, you are reverting back my changes. Please create new header constants if required.